llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -2518,7 +2518,7 @@ bool SelectionDAG::expandMultipleResultFPLibCall(
   unsigned NumResults = Node->getNumValues();
 
   const char *LCName = TLI->getLibcallName(LC);
-  if (!LC || !LCName)
+  if (!LCName)
```

Comment:
It should include an explicit check against the invalid enum value (which I was also planning on moving to be 0). I have a pending patch which replaces most of this with checks against the LibcallImpl anyway